api/api.py

```diff
@@ -2,15 +2,16 @@
 from functools import reduce
 
 from django.conf import settings
-from django.db.models import F, Q
+from django.db.models import F, Q, Sum
 from rest_framework import serializers
 from rest_framework.decorators import action
 from rest_framework.response import Response
 from rest_framework.viewsets import ReadOnlyModelViewSet
 
 from apps.budget.choices import ExpenseGroupChoices, RevenueGroupChoices
-from apps.budget.models import Budget, TransparencyIndex, Expense, Revenue, BudgetSummary, Category, CategoryMap
+from apps.budget.models import Budget, TransparencyIndex, Expense, Revenue, BudgetSummary, Category, CategoryMap, budget
```

Comment:
Check if 'budget' being imported is in use. Imports from models should always import Models.